{"name": "test_case[case1]", "status": "broken", "statusDetails": {"message": "TypeError: 'tuple' object is not callable", "trace": "self = <test_case.Test_Case object at 0x00000255DA20DC88>\ncase = {'case_description': 'get /topic/:id 主题详情', 'case_expect': '{\"success\":true}', 'case_method': 'get', 'case_params': '', ...}\n\n    @pytest.mark.parametrize('case',Excel_Handler().get_excel_data())\n    def test_case(self,case):\n        print(11111111111111111,case)\n>       response = RequestHandler(case).get_response()\nE       TypeError: 'tuple' object is not callable\n\nscripts\\test_case.py:14: TypeError"}, "parameters": [{"name": "case", "value": "{'case_project': 'cnodejs项目', 'case_description': 'get /topic/:id 主题详情', 'case_url': 'https://cnodejs.org/api/v1/topic/5433d5e4e737cbe96dcef312', 'case_method': 'get', 'case_params': '', 'case_expect': '{\"success\":true}', 'other': ''}"}], "start": 1574774792943, "stop": 1574774793631, "uuid": "ce7b98c3-ef7e-4c02-a41f-01a874b4c466", "historyId": "a01bd1db2b036dec6c3e0f709de8a2b1", "testCaseId": "89a27520489a549f0caeaffa7948244d", "fullName": "scripts.test_case.Test_Case#test_case", "labels": [{"name": "parentSuite", "value": "scripts"}, {"name": "suite", "value": "test_case"}, {"name": "subSuite", "value": "Test_Case"}, {"name": "host", "value": "IG6I7Y1F5I7TZH5"}, {"name": "thread", "value": "15128-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "scripts.test_case"}]}